node-proxy/openshift-origin-node-proxy.spec

```diff
@@ -132,6 +132,7 @@ fi
 %endif
 %attr(0755,-,-) %{_bindir}/node-find-proxy-route-files
 %attr(0640,-,-) %{_sysconfdir}/openshift/web-proxy-config.json
+%config(noreplace) %{_sysconfdir}/openshift/web-proxy-config.json
```

Comment:
Documentation implies that the configuration file will be overwritten if no changes has been made. I assume it is the same when adding `%config` to a file, but can't find a definitive answer.

```
If you use %config(noreplace), the file will not overwrite an existing file that has been modified. If the file has not been modified on disk, the rpm command will overwrite the file. But, if the file has been modified on disk, the rpm command will copy the new file with an extra file-name extension of .rpmnew. 
```
https://docs.fedoraproject.org/ro/Fedora_Draft_Documentation/0.1/html/RPM_Guide/ch09s05s03.html

In testing, I modified the configuration file and updated to the rpm that includes these changes. The configuration file was not touched. A `.rpmnew` file was created containing the new `ciphers` option.